### DISCUSSION TITLE
Can you adjust the update of the raycast?

### DISCUSSION THREAD

**Discussion by imjusthere** [microsoft/MixedRealityToolkit-Unity#11554] [Q&A]
I'm using LeanTween to move the XR Rig and when I do that the far ray bends with the movement of the XR Rig. I think this is happening because the update of the tween doesn't match the ray cast update. Any ideas on how to fix this? The LeanTween appears to be happening on Update. I've tried changing a lot of the XR Rig settings but nothing has made any affect on this issue yet. Thoughts?

Background: I made a VR subway. I'm using LeanTween to move the subway cars. If the player is on the train then the XR Rig gets parented to the subway car. That is when this issue happens. When the cars move with the player on it the raycast gets dragged. I really like using the tween engine because the eases are perfect for the train movement.

UPDATE: Nevermind. I changed the order of the leantween scripts using the "Script Execution Order" in the project settings and now it works. Leantween had to run before the MRTK3 code.

**Comment by AMollis** [comment]
@imjusthere closing discussion since you've seemed to have answered it :)
